test/index.js

```diff
@@ -1623,32 +1623,20 @@ describe('function helpers', () => {
 describe('api route file analysis', () => {
   it('extracts correct route configs from source files', async () => {
     await moveNextDist()
-    const configs = await getApiRouteConfigs('.next', process.cwd())
+    const configs = await getAdvancedApiRouteConfigs('.next', process.cwd())
     // Using a Set means the order doesn't matter
     expect(new Set(configs)).toEqual(
       new Set([
-        { compiled: 'pages/api/enterPreview.js', config: {}, route: '/api/enterPreview' },
```

Comment:
Only generate custom rewrites for advanced API routes (background and scheduled functions).